README.md

```diff
@@ -75,14 +74,6 @@ This repository includes files with plain SQL that can be used to recreate a dat
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
-
-## 🚀 Live Demo <a name="live-demo"></a>
-
-- [Under construction](#)
-
```

Comment:
- [ ] Nice job so far… please your MIT license like is broken.. kindly fix that to keep your readme file professional and clean

Reply:
**Hi @tufoinnkuo10 👋,**
Thank you so much 🙏  for reviewing my project.
Your nice feedback is appreciated 🙏.
I have checked the issue and made it fix 🎯.